curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/uncomment-html.english.md

```diff
@@ -23,14 +23,14 @@ Uncomment your <code>h1</code>, <code>h2</code> and <code>p</code> elements.
 
 ```yml
 tests:
-  - text: Your <code>h1</code> element should be visible on your page by uncommenting it.
+  - text: Your <code>h1</code> element should be visible on the page by uncommenting it.
     testString: assert($("h1").length > 0);
-  - text: Your <code>h2</code> element should be visible on your page by uncommenting it.
+  - text: Your <code>h2</code> element should be visible on the page by uncommenting it.
     testString: assert($("h2").length > 0);
-  - text: Your <code>p</code> element should be visible on your page by uncommenting it.
+  - text: Your <code>p</code> element should be visible on the page by uncommenting it.
     testString: assert($("p").length > 0);
-  - text: All trailing comment tags should be deleted&#44; i.e. <code>--&#62;</code>.
-    testString: assert(!/[^fc]-->/gi.test(code.replace(/ *<!--[^fc]*\n/g,'')));
+  - text: No trailing comment tags should be visible on the page (i.e. <code>--></code>).
+    testString: console.log($('*:contains("-->")')[1]);assert(!$('*:contains("-->")')[1]);
```

Comment:
```suggestion
    testString: assert(!$('*:contains("-->")')[1]);
```
I'm assuming you just forgot to take that out? I'm not sure what this test is doing - but it looks like it works, so I'm fine with it.

Reply:
The assert just makes sure someone did not forget to remove `-->` after removing the `<!--`.  If only the `<!--` was removed, the `-->` would be visible on the page (which is what the jQuery expression is checking).